appium/lib/screen-object.rb

```diff
@@ -37,9 +37,6 @@ def driver
     ScreenObject::AppElements::Element.new('').driver
   end
 
-  def swipe(start_x,start_y,end_x,end_y,touch_count,duration)
-    driver.swipe(:start_x => start_x, :start_y => start_y, :end_x => end_x, :end_y => end_y,:touchCount => touch_count,:duration => duration)
```

Comment:
Undelete `swipe` and update it to:

```
def swipe(start_x, start_y, end_x, end_y, duration)
  gesture(start_x, start_y, end_x, end_y, duration)
end
```

Reply:
We could also alias the method.

Reply:
Also, did we ever figure out what was wrong with `driver.swipe` method of Appium? We should put a comment on why we are using `gesture` instead of Appium's built-in `swipe`

Reply:
I just noticed that you are using `swipe` only in `gesture`. I would say just move that logic to this method because `swipe` seems like a better name for a screen object method that `gesture`